Libraries/Lists/MetroListView.js

```diff
@@ -189,10 +192,24 @@ class MetroListView extends React.Component<Props, $FlowFixMeState> {
       };
     }
   }
+  /* $FlowFixMe(>=0.53.0 site=react_native_fb,react_native_oss) This comment
+   * suppresses an error when upgrading Flow's support for React. To see the
+   * error delete this comment and run Flow. */
+  _renderEmpty = () => <this.props.ListEmptyComponent key="$empty" />;
   /* $FlowFixMe(>=0.53.0 site=react_native_fb,react_native_oss) This comment
    * suppresses an error when upgrading Flow's support for React. To see the
    * error delete this comment and run Flow. */
   _renderFooter = () => <this.props.FooterComponent key="$footer" />;
+  /* $FlowFixMe(>=0.53.0 site=react_native_fb,react_native_oss) This comment
+   * suppresses an error when upgrading Flow's support for React. To see the
+   * error delete this comment and run Flow. */
```

Comment:
Do we need to add keys here to avoid the missing key warning?

Reply:
There are keys in the _renderHeader and _renderEmpty. Is this not sufficient?

Reply:
Ohh I see, yea this is good!